components/clarinet-sdk/node/tests/remote-data.test.ts

```diff
@@ -0,0 +1,70 @@
+import fs from "node:fs";
+import path from "node:path";
+import { Cl } from "@stacks/transactions";
+import { describe, expect, it, beforeEach, afterEach } from "vitest";
+
+// test the built package and not the source code
+// makes it simpler to handle wasm build
+import { getSDK } from "..";
+
+const api_url = "https://api.testnet.hiro.so";
+const counterAddress = "STJCAB2T9TR2EJM7YS4DM2CGBBVTF7BV237Y8KNV.counter";
+const sender = "ST2CY5V39NHDPWSXMW9QDT3HC3GD6Q6XX4CFRK9AG";
+
+const deploymentPlanPath = path.join(
+  process.cwd(),
+  "tests/fixtures/deployments/default.simnet-plan.yaml",
+);
+
+function deleteExistingDeploymentPlan() {
+  if (fs.existsSync(deploymentPlanPath)) {
+    fs.unlinkSync(deploymentPlanPath);
+  }
+}
+
+beforeEach(async () => {
+  deleteExistingDeploymentPlan();
+});
+
+afterEach(() => {
+  deleteExistingDeploymentPlan();
+});
+
+describe("simnet remote interactions", async () => {
+  const simnet = await getSDK();
+
+  it("can call a remote contract", async () => {
+    await simnet.initEmptySession({
+      enabled: true,
+      api_url: "https://api.testnet.hiro.so",
```

Comment:
In this PR we are running some tests again testnet... Which doesn't sounds like a very good idea. But it is convenient.
Other options would be to
- test against a local devnet (but we know that's annoying to do in github actions)
- test against mainnet might actually be better, since tesnet can be reset at anytime